capi/VideoDecoderCapi.cpp

```diff
@@ -18,8 +18,8 @@
 #include "config.h"
 #endif
 #include "VideoDecoderCapi.h"
-#include "interface/VideoDecoderInterface.h"
-#include "interface/VideoDecoderHost.h"
+#include "VideoDecoderInterface.h"
+#include "VideoDecoderHost.h"
```

Comment:
Ok, but we still need `-I$(top_srcdir)/interface \` in makefile.am. else it not build able, right?